src/Scoringline/SendinblueApi/Api/Email.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * This file is a part of scoringline sendinblue api package
+ *
+ * (c) Scoringline <joni@sendinblue.com>
```

Comment:
Don't change the copyright. If you want to add your email, you can add a docblock on the class with `@author Joni Rajput <joni@sendinblue.com>`.